react/components/ProductImages/components/Carousel/index.js

```diff
@@ -259,7 +259,7 @@ class Carousel extends Component {
             bulletSelector: `.swiper-pagination-bullet`,
             bulletClass: `swiper-pagination-bullet ${cssHandles.swiperBullet}`,
             bulletActiveClass:
-              'c-action-primary swiper-pagination-bullet-active',
+              `c-action-primary swiper-pagination-bullet-active ${applyModifiers(cssHandles.swiperBullet), 'isActive'}`,
```

Comment:
```suggestion
              `c-action-primary swiper-pagination-bullet-active ${applyModifiers(cssHandles.swiperBullet, 'isActive')}`,
```

Also, you have to fix the lint problems, to do that you can run yarn in the root project and the eslint errors should appear